ui-v2/app/controllers/dc/nspaces/create.js

```diff
@@ -0,0 +1,2 @@
+import Controller from './edit';
+export default Controller.extend();
```

Comment:
If you're not doing anything here, you can remove this file and ember will create it automatically for you. No harm in leaving it either, just thought I'd mention it.

Reply:
Yeah I think we'd usually delete these, but this one needs to be here in order to extend `./edit`

Reply:
Ha thanks, I totally missed that that's what was happening, that answers another question I had too.